src/index.html

```diff
@@ -7,8 +7,65 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="logo-image"
+        >
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__item-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__item-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones"class="nav__item-link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops"
+              data-qa="hover"
+              class="nav__item-link"
+            >
+                Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets"class="nav__item-link">Gadgets</a>
```

Comment:
Use spaces between att names
```suggestion
            <a href="#Gadgets" class="nav__item-link">Gadgets</a>
```